modules/RTC/TraceablePeerConnection.js

```diff
@@ -1536,6 +1536,17 @@ TraceablePeerConnection.prototype.setLocalDescription
 
     this.trace('setLocalDescription::preTransform', dumpSDP(localSdp));
 
+    if (this.options.preferH264) {
```

Comment:
Do you think it's worth it to extract this to an auxiliary function and use it on both local and remote?